src/components/CollapsibleTable.js

```diff
@@ -145,9 +164,16 @@ const CollapsibleRow = (props) => {
           </IconButton>
         </StyledTableCell>
         {row.data.map((cell, i) => (
-          <StyledTableCell key={i}>
-            {cell.value}
-          </StyledTableCell>
+          cell.truncate ?
```

Comment:
Would optional chaining work here and allow for fewer changes in `DatasetSearchTable.js`?
```suggestion
          cell?.truncate ?
```